chapters/arrays.tex

```diff
@@ -1263,7 +1263,7 @@ \subsection{Element-wise Division}\label{array-element-wise-division}\label{elem
 2./[1, 2; 3, 4]  // error; same as  2.0 / [1, 2; 3, 4]
 2 ./[1, 2; 3, 4] // fine; element-wise division
 \end{lstlisting}
-This is a consequence of the parsing rules, since `\lstinline!2.!' is a lexical unit.  Using a space after the literal solves the problem.
+This is a consequence of the parsing rules, since ``\lstinline!2.!'' is a lexical unit.  Using a space after the literal solves the problem.
```

Comment:
I would prefer to just have single quotes for short code fragments, regardless of whether they are one character or a few - i.e. skip this.

Reply:
I'm not against it.  You do realize that his means there are a few other places where we'll then need to change from double to single quotes?

Reply:
Yes. But better to have one rule, and similarly as we say that we can add the quotes later when needed we can also clean up that later.

Reply:
OK now?